code/datums/status_effects/debuffs.dm

```diff
@@ -182,7 +182,9 @@
 		marked_underlay.pixel_x = -owner.pixel_x
 		marked_underlay.pixel_y = -owner.pixel_y
 		owner.underlays += marked_underlay
-	return FALSE
+		return TRUE
+	else 
```

Comment:
That's true, but I thought it made the code look nicer - but since you disagree I'll remove it